lib/limiter/internal/ratelimit/http.go

```diff
@@ -0,0 +1,59 @@
+// Teleport
+// Copyright (C) 2024 Gravitational, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
```

Comment:
I don't know if we're allowed to relicense code from oxy, it's a fork of vulcand/oxy that has had many contributors, even if the ratelimit package seems to be almost entirely from gravitational.

Perhaps just to be safe we should keep the Apache license?

Reply:
This particular file is new code, but I'll think more about the other files.